spec/requests/webhooks_spec.rb

```diff
@@ -24,7 +24,51 @@
         line_items: line_items
       )
     end
-    let(:seller) { create :seller }
+    let!(:seller_1) do
+      Seller.create(
```

Comment:
You should use the Seller factory `create :seller, seller_id: 'shunfa-bakery', ...` but you probably don't need to add the rest of the stuff since it'll be randomized

Reply:
Gotcha yay!